### PR TITLE
Fix a couple of npc locations

### LINKED ISSUE #504: NPC incorrect location Opus
**Current behavior**: 
Opus is in incorrect location

**Client version**: 
1.12.1
**Commit hash**: 

**Operating system**: 
Win 10

### PR DESCRIPTION
**Pull request by NickTyrer**
fixes #504 and #508

### CODE REVIEW

File: sql/migrations/20200504144725_world.sql [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+BEGIN
+DECLARE v INT DEFAULT 1;
+SET v = (SELECT COUNT(*) FROM `migrations` WHERE `id`='20200504144725');
+IF v=0 THEN
+INSERT INTO `migrations` VALUES ('20200504144725');
+-- Add your query below.
+
+
+-- Alowicious Czervik correct location
+UPDATE `creature` SET `position_x` = 1660.02, `position_y` = -4391.0, `position_z` = 23.2680, `orientation` = 4.771000 WHERE `guid` = 86237;
```
Comment (ratkosrb) [inline sql/migrations/20200504144725_world.sql]: That is the same location he has currently.

File: sql/migrations/20200504144725_world.sql [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+-- Opus correct location
+UPDATE `creature` SET `position_x` = -7468.840820, `position_y` = -2186.785156, `position_z` = 164.418686, `orientation` = 2.294904, `movement_type` = 0, `wander_distance` = 0 WHERE `guid` = 4567;
```
Comment (ratkosrb) [inline sql/migrations/20200504144725_world.sql]: I see several different positions for this npc in sniff, so why are you removing the waypoints? He must be moving.

-7463.3 -2187.77 167.351 2.69095
-7477.56 -2188.48 165.374  3.08992
-7461.74 -2188.44 167.438 2.57953
-7461.38  -2188.6 167.459 2.59347
-7469.25  -2186.32 164.268 2.21123

These are coordinates i have him at.

Comment (NickTyrer) [reply sql/migrations/20200504144725_world.sql]: He definitely does not move, and his current position in vmangos is incorrect. Maybe he has different spawn points?
https://youtu.be/5soMRrt-3e4

**Comment by NickTyrer** [comment]
@ratkosrb what changes do you need here to make this mergable?

**Comment by ratkosrb** [comment]
Need to figure out why we have multiple positions for Opus. Maybe his movement script is bugged in Classic and he gets stuck there. Should visit him right after a server restart to check if he is in the same spot.

**Comment by GrenderG** [comment]
Fun fact: Seems like he was called **Toucan Stan** before being called Opus: https://web.archive.org/web/20050206205148/http://wow.allakhazam.com/db/mob.html?wmob=10262

**Comment by Daribon** [comment]
@GrenderG Looks like they forgot to change it in the quest progress for Kibler's Exotic Pets, they probably fixed it in some expansion: https://vanillawowdb.com/?quest=4729

**Progress**
Kibler is busy arguing with Toucan Stan.

Since classic has a mix of expansion fixes in it, in classic its showing as:
Kibler is busy arguing with Opus.

OR they changed Toucan Stan to Opus in some later patch of vanilla and when private server DBs were created (which vanillawowdb.com is based on) the mob was still named Toucan Stan and the quest text used Toucan Stan.

**Comment by mikemenasi** [comment]
@ratkosrb after a server restart he spawned in a different location with a different model. No movement though

**Comment by NickTyrer** [comment]
@ratkosrb I have removed the fix for opus from this PR to move it forward.

**Pull request merged**